doc/src/reference.xml

```diff
@@ -686,8 +688,8 @@ path-constant DATA : data/a.txt ;
         <listitem>
           <para><emphasis role="bold">Allowed values:</emphasis> <literal>on</literal>, <literal>off</literal>.</para>
 
-          <para>The <literal>runtime-debugging</literal> feature specifies if
-          produced object files, executables and libraries should include
+          <para>The <literal>runtime-debugging</literal> feature specifies that,
+          if produced, object files, executables, and libraries should include
```

Comment:
s/if/whether/ would be a more accurate rendering of the original intent of this sentence.